builder/amazon/chroot/builder.go

```diff
@@ -501,3 +536,22 @@ func (b *Builder) Run(ctx context.Context, ui packer.Ui, hook packer.Hook) (pack
 
 	return artifact, nil
 }
+
+func validateKmsKey(kmsKey string) (valid bool) {
```

Comment:
Looks like this was copied from `common/ami_config.go`. How about we just change the func in `common` to be exported (all we should need to do is capitalize it) and then reuse it in the chroot builder as `awscommon.ValidateKmsKey`?

Reply:
lol forgot about the copy/paste of that function to make it work. Will certainly do, thanks for the review : D

Reply:
@SwampDragons Done, I've removed my copy/paste and exported the function on `common/ami_config.go`

Let me know if there is anything else : )